### PR TITLE
[Keymap] Add Dactyl Manuform 5x6 with 5 thumb keys

### PR DESCRIPTION
**Pull request by jceb**

### CODE REVIEW

**Comment by jceb** [comment]
The PR is pending: https://github.com/qmk/qmk_firmware/pull/9659

**Comment by jceb** [comment]
@noroadsleft PR https://github.com/qmk/qmk_firmware/pull/9659 went through. Would you merge this PR?

BTW, I'm using a `keymap.json` in the qmk_firmware repository for this keyboard. Could this be used by qmk_configurator in the future?

File: public/keymaps/h/handwired_dactyl_manuform_5x6_5_default.json [hunk 1]
```diff
@@ -0,0 +1,507 @@
+{
+    "version": 1,
+    "notes": "",
+    "author": "Jan Christoph Ebersbach",
+    "keyboard": "handwired/dactyl_manuform/5x6_5",
+    "keymap": "default",
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_dactyl_manuform_5x6_5_default.json:6]:
```suggestion
    "commit": "1a49eb5be12fd5a32d59103427c71ee4869eea49",
    "keymap": "default",
```

**Comment by noroadsleft** [comment]
> BTW, I'm using a `keymap.json` in the qmk_firmware repository for this keyboard. Could this be used by qmk_configurator in the future?

That's the long-term goal, actually, but I believe we're quite a ways away from that happening.

**Comment by noroadsleft** [comment]
... and then I caused you a merge conflict because I forgot I had this file on another branch that I just merged yesterday... :man_facepalming:

**Comment by noroadsleft** [comment]
... and fixed. Thanks!

**Pull request merged**